sdk/communication/tests.yml

```diff
@@ -0,0 +1,30 @@
+trigger: none
+
+parameters:
+  - name: TestPackagesEnabled # by overriding this parameter, developers can control what tests to run
```

Comment:
We have gotten rid of the TestPackagesEnabled parameter. Please refer to the latest tests.yml in other communication SDKs and update this.

Reply:
Updated